frontend/apps/reader/modules/readerstatus.lua

```diff
@@ -38,6 +39,9 @@ function ReaderStatus:addToMainMenu(menu_items)
 end
 
 function ReaderStatus:onEndOfBook()
+    if Device:isHapticFeedbackEnabled() then
```

Comment:
I think it's probably more elegant if this just reads `Device:performHapticFeedback("CONTEXT_CLICK")`.

Reply:
You mean fold the cap check inside performHaptic*?

Reply:
Exactly. It already defaults as a no-op anyway and this way it just adds a lot of extra boilerplate.